src/apps/HeyGPT/HeyGPT/ViewModels/LetsChatViewModel.cs

```diff
@@ -120,6 +137,74 @@ private void OnExecuteToggleShouldIncludeClipboardContextCommand()
 
     #endregion
 
+    #region Event Handlers
+
+    private void SubscribeToEvents()
+    {
+        _eventAggregator.Subscribe<FileActivatedEvent>(OnFileActivatedEvent);
+        _eventAggregator.Subscribe<ProtocolActivatedEvent>(OnProtocolActivatedEvent);
+    }
+
+    private void OnFileActivatedEvent(FileActivatedEvent fileActivatedEvent)
+    {
+        var firstFile = fileActivatedEvent.Files[0];
```

Comment:
When you double click on an *.heygpt file you will arrive here.   This code will attempt to read the file specified and extract a message prompt.  We support both a structured and unstructured content.   The structured is a JSON array of SimpleMessagePrompt.   We chose one of the items in the file at random.   The unstructured will just be text and will be sent as is.

Reply:
We really need to add some input validation here to ensure we aren't sending the Internet.   The token limit is 1000